dockertest/environment.py

```diff
@@ -49,10 +46,10 @@ def get_selinux_context(path):
     :return: SELinux context as a string or None if not set
     """
     try:
-        # may include null-termination in string
-        return getxattr(path, 'security.selinux').replace('\000', '')
+        # First list item is null-terminated string length
+        return selinux.getfilecon(path)[1]
     except IOError, xcpt:
-        if xcpt.errno == errno.EOPNOTSUPP:
+        if xcpt.errno == errno.ENOENT:
```

Comment:
> libselinux raises a different exception.

When? What is causing that exception? In reading that, what I see is "if file doesn't exist, i.e. caller has fed us garbage, return None". That doesn't seem meaningful. Is selinux raising ENOENT on some other condition?